analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/EmptyNullableValueAccessTest.cs

```diff
@@ -18,47 +18,68 @@
+        .WithBasePath(@"SymbolicExecution\Sonar")
+        .WithOnlyDiagnostics(ChecksCS.EmptyNullableValueAccess.S3655);
+
+    private readonly VerifierBuilder roslynCS = new VerifierBuilder()
+        .AddAnalyzer(() => new CS.SymbolicExecutionRunner(AnalyzerConfiguration.AlwaysEnabled))
+        .WithBasePath(@"SymbolicExecution\Roslyn")
+        .WithOnlyDiagnostics(ChecksCS.EmptyNullableValueAccess.S3655);
+
+    [DataTestMethod]
+    [DataRow(ProjectType.Product)]
+    [DataRow(ProjectType.Test)]
+    public void EmptyNullableValueAccess_Sonar_CSharp8(ProjectType projectType) =>
+        sonar.AddPaths("EmptyNullableValueAccess.cs")
+            .AddReferences(TestHelper.ProjectTypeReference(projectType).Concat(MetadataReferenceFacade.NETStandard21))
+            .WithOptions(ParseOptionsHelper.FromCSharp8)
+            .Verify();
 
-        [DataTestMethod]
-        [DataRow(ProjectType.Product)]
-        [DataRow(ProjectType.Test)]
-        public void EmptyNullableValueAccess_CSharp8(ProjectType projectType) =>
-            sonar.AddPaths("EmptyNullableValueAccess.cs")
-                .AddReferences(TestHelper.ProjectTypeReference(projectType).Concat(MetadataReferenceFacade.NETStandard21))
-                .WithOptions(ParseOptionsHelper.FromCSharp8)
-                .Verify();
+    [DataTestMethod]
+    [DataRow(ProjectType.Product)]
+    [DataRow(ProjectType.Test)]
+    public void EmptyNullableValueAccess_Roslyn_CSharp8(ProjectType projectType) =>
+        roslynCS.AddPaths("EmptyNullableValueAccess.cs")
+            .AddReferences(TestHelper.ProjectTypeReference(projectType).Concat(MetadataReferenceFacade.NETStandard21))
+            .WithOptions(ParseOptionsHelper.FromCSharp8)
+            .Verify();
 
 #if NET
 
-        [TestMethod]
-        public void EmptyNullableValueAccess_CSharp9() =>
-            sonar.AddPaths("EmptyNullableValueAccess.CSharp9.cs")
-                .WithTopLevelStatements()
-                .Verify();
+    [TestMethod]
+    public void EmptyNullableValueAccess_Sonar_CSharp9() =>
+        sonar.AddPaths("EmptyNullableValueAccess.CSharp9.cs").WithTopLevelStatements().Verify();
 
-        [TestMethod]
-        public void EmptyNullableValueAccess_CSharp10() =>
-            sonar.AddPaths("EmptyNullableValueAccess.CSharp10.cs")
-                .WithOptions(ParseOptionsHelper.FromCSharp10)
-                .Verify();
+    [TestMethod]
+    public void EmptyNullableValueAccess_Roslyn_CSharp9() =>
+        roslynCS.AddPaths("EmptyNullableValueAccess.CSharp9.cs").WithTopLevelStatements().Verify();
+
+    [TestMethod]
+    public void EmptyNullableValueAccess_Sonar_CSharp10() =>
+        sonar.AddPaths("EmptyNullableValueAccess.CSharp10.cs").WithOptions(ParseOptionsHelper.FromCSharp10).Verify();
+
+    [TestMethod]
+    public void EmptyNullableValueAccess_Roslyn_CSharp10() =>
+        roslynCS.AddPaths("EmptyNullableValueAccess.CSharp10.cs").WithOptions(ParseOptionsHelper.FromCSharp10).Verify();
```

Comment:
Each call should be in a separate lines when chaining methods.

Reply:
No, this is fine when it fits on a single line without violating the too long line rule.